xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.h

```diff
@@ -8,12 +8,23 @@
 
 #pragma once
 
+#include "SubtitlesStyle.h"
 #include "threads/CriticalSection.h"
+#include "utils/ColorUtils.h"
 
 #include <ass/ass.h>
+#include <ass/ass_types.h>
 
 /** Wrapper for Libass **/
 
+static constexpr int ASS_NO_ID = -1;
+
+enum ASSSubType
```

Comment:
Still needed? With different code paths and different overlay types (text/adapted) and ass (native) does the handler still need to know the context of the call?

Reply:
if you remove it you need to add a compare to check each overlay, has no sense,
IMO is better kept as is to be set just after libass Configure method
this can be useful also for possible changes in future libass versions.
if you do not want an enum i could change it with a bool like:
`bool m_isNativeAssType;` and related set method
but to me seem more clean as is